packages/amplify-category-geo/src/index.ts

```diff
@@ -0,0 +1,19 @@
+import path from 'path';
+import { category } from './constants';
+
+export async function executeAmplifyCommand(context: any) {
```

Comment:
use const arrow notation for defining functions: `export const funcName = () => {}`

(applies throughout the PR)